### PR TITLE
Clarify how to verify the APT source

### PR DESCRIPTION
**Pull request by mstanleyjones**
### Describe the proposed changes

- Expand information about how to verify that your APT sources are correct
- Clean up some formatting along the way

### Project version

n/a

### Related issue

#167 

### Related issue or PR in another project

n/a

### Please take a look

@thaJeztah @grantwinney

### CODE REVIEW

File: engine/installation/linux/ubuntulinux.md [hunk 1]
```diff
@@ -49,64 +49,90 @@ your `APT` sources to the new Docker repository.
...
 
-4. Add the new `GPG` key.
+    ```bash
+    $ sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
+    ```
 
-        $ sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
-
-5. Open the `/etc/apt/sources.list.d/docker.list` file in your favorite editor.
+5.  Open the `/etc/apt/sources.list.d/docker.list` file in a text editor.
```
Comment (thaJeztah) [inline engine/installation/linux/ubuntulinux.md]: for a follow up, we could consider just showing `echo deb ...... > /etc/apt/sources.list.d/docker.list`

File: engine/installation/linux/ubuntulinux.md [hunk 2]
```diff
@@ -49,64 +49,90 @@ your `APT` sources to the new Docker repository.
...
-        $ sudo apt-get update
+    ```bash
+    $ sudo apt-get update
+    ```
 
 10. Purge the old repo if it exists.
 
-        $ sudo apt-get purge lxc-docker
+    ```bash
+    $ sudo apt-get purge lxc-docker
```
Comment (thaJeztah) [inline engine/installation/linux/ubuntulinux.md]: We should check if this is still an issue, `lxc-docker` is really ancient now, so not very likely that people run into this

Comment (mstanleyjones) [inline engine/installation/linux/ubuntulinux.md]: It's not an issue anymore in 14.04 and later. I'll take it out.

**Comment by mstanleyjones** [comment]
I made the two suggestions, and it shortened the procedure by several steps! LMK what you think.

**Pull request merged**